.github/labeler.yml

```diff
@@ -1,5 +1,5 @@
 chore:
-  - any: ['*']
+  - any: ['*', '**/*']
```

Comment:
```suggestion
  - any: ['*', '**/**']
```
🤔 ?

Reply:
nope, **/* means for example folder/folder/folder/hello.ts